jujugui/static/gui/src/app/components/deployment-flow/test-deployment-flow.js

```diff
@@ -23,118 +23,113 @@ var juju = {components: {}}; // eslint-disable-line no-unused-vars
 chai.config.includeStack = true;
 chai.config.truncateThreshold = 0;
 
+/**
+  Convenience function for quickly and easily creating a DeploymentFlow
+  component. It accept a props param, which can be used to change defaults (on
+  required props) or provide values (on optional props).
+ */
+const createDeploymentFlow = (props = {}) => {
```

Comment:
Hmmm... I'm not sure about this, I think the scoping will introduce a bunch of new problems. Happy to be convinced otherwise... maybe we should hold off on big changes like this to our tests until we have a discussion about testing next week.